mantidproject/mantid
# Add fitting tab outline to Muon Analysis 2

**Description of work.**

This is step one of adding a fitting tab to to the new Muon interfaces. This is currently a work in progress so not all of the expected functionality will be in this PR. The Muon Analysis interface is currently hidden from users however so this should not be an issue. 

**To test:**

Functionality added here which should be tested:

* Load some data and switch to the fitting tab, check it has chosen a sensible default for the workspace to fit.
* Click on the select data to fit button and a dialog window allowing you to select from current group, pair and phase-quad work spaces should be shown. Select some work spaces and re-order them via drag and drop and check that the correct workspaces in the correct order and shown once you close the dialog box.






Partially Fixes #25585 



*This does not require release notes* because this interface is not yet visible to users

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
